.github/workflows/docs.yml

```diff
@@ -0,0 +1,51 @@
+name: AMR-Wind Docs
+
+ on:
```

Comment:
@jrood-nrel so there is a syntax error in the file... this node and subsequent nodes are offset by 1 space. So it appears as a sub-dictionary of `name` and that's why I think the workflow didn't run when you submitted the PR.

Reply:
Ah, damn. Copy and paste foiled me.